moto/kms/models.py

```diff
@@ -348,6 +379,16 @@ def describe_key(self, key_id: str) -> Key:
         key_id = self.get_key_id(key_id)
         if r"alias/" in str(key_id).lower():
             key_id = self.get_key_id_from_alias(key_id)  # type: ignore[assignment]
+
+        key = self.keys[self.get_key_id(key_id)]
+
+        if key.multi_region:
+            if key.arn == key.multi_region_configuration["PrimaryKey"]["Arn"]:
+                return self.keys[self.get_key_id(key_id)]
+            else:
+                key.multi_region_configuration["MultiRegionKeyType"] = "REPLICA"
```

Comment:
Can we move this to the `replicate_key`-method? That's where we decide that a given key is a replica, so it makes sense to me to set the KeyType as well.

Reply:
Ah, yes, I understand - it makes sense as it is then.